1-js/99-js-misc/06-unicode/article.md

```diff
@@ -1,172 +1,167 @@
 
-# Unicode, String internals
+# Unicode, detalhes internos da String
 
-```warn header="Advanced knowledge"
-The section goes deeper into string internals. This knowledge will be useful for you if you plan to deal with emoji, rare mathematical or hieroglyphic characters, or other rare symbols.
+```warn header="Conhecimento avançado"
+Esta seção aprofunda nos detalhes internos das strings. Este conhecimento será útil se você planeja lidar com emojis, caracteres matemáticos raros, hieróglifos ou outros símbolos raros.
```

Comment:
Eu odeio a palavra "hieróglifos" usada em diversos trechos que tratam de Unicode, sinto que é um pouco xenofóbico. Eu sempre traduzo por "logogramas" ou outros nomes mais apropriados.

Reply:
@Peruibeloko, não é xenofóbico, é um termo muito usado na arqueologia.
Recomendo as pessoas a entenderem as coisas e dos porquês (se existirem) antes de partilharem opiniões sobre assuntos fora do seu domínio, mas geralmente se está fora do seu domínio não dê opiniões.

Reply:
@nazarepiedady sim, é um termo muito usado na arqueologia, mas nos trechos que me refiro, o autor não usa a palavra nesse sentido.

Repare que é possível ver pelo contexto, não só nesse artigo mas em outros também, que o terno hieróglifos está sendo usado no lugar de logogramas. Hieróglifos são especificamente os usados pelos egípcios antigos, logogramas são qualquer sistema de escrita que usa um único caractere para representar um morfema, palavra ou conceito.

Hieróglifos são um tipo de logograma, mais especificamente ideograma, já que usam representações visuais dos conceitos.

Reply:
@Peruibeloko entendo, obrigado pelo esclarecimento!
Mas agora, @nazarepiedady e @Peruibeloko, o que não ficou muito claro pra mim é: mantenho como está ou faço a alteração para "logograma"?

Reply:
Keep it as it is.